### DISCUSSION TITLE
Menambahkan privilege di Shinyapps

### DISCUSSION THREAD

**Discussion by shinddayo** [teamalgoritma/community#777] [Q&A] [answered]
Bagaimanakah menambahkan privilege di Shinyapps, sehingga hanya orang tertentu/terdaftar yang bisa buka report nya?

**Comment by ramdhinafinita** [comment, marked as answer]
Dear @shinddayo 
Bisa dilakukan hanya saja membutuhkan pembelian akun, karena yang biasa hanya free jadi setiap publish pasti semua orang bisa melihat.
 Lalu pertanyaanya
, apakah Shiny bisa dibuat private?

1. Opsi 1: Subsribe shiny untuk membuka fiture private, cm ini fiture yang profesional (paling mahal/professional) https://www.shinyapps.io/
![image](https://github.com/teamalgoritma/community/assets/64061969/aa30b6a8-6fef-4e93-9f45-0052659c06d0)

2. Opsi 2: Membuat ID & Password di shiny free, [referensi](https://www.listendata.com/2019/06/how-to-add-login-page-in-shiny-r.html)

 Jika jawaban dari pertanyaan yang diajukan bermanfaat dan dapat diimplementasi silahkan lakukan "Mark as answer" pada jawaban ya pak, terimakasih
